### PR TITLE
[test] quality preload

### PR DESCRIPTION
**Pull request by oded-soffrin**
just to test e2e with experiment "turned on"

### CODE REVIEW

**Comment by oded-soffrin** [comment]
image
Found a case where qualtiy experiment doesn't work, need to be fixed before we open the experiment

**Comment by oded-soffrin** [comment]
Solved! all test fixtures looks the same (+- a few pixels)
@justame FYI!

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 1]
```diff
@@ -111,10 +112,27 @@ class ImageViewer extends React.Component {
...
-      imageUrl.preload = getImageSrc(src, helpers, {
-        ...(useQualityPreoad && { imageType: 'quailtyPreload' }),
-      });
+      const {
+        componentData: {
+          config: { alignment, width },
+        },
+      } = this.props;
+
+      const useQualityPreoad = true; //this.context.experiments?.useQualityPreoad?.enabled;
```
Comment (justame) [inline packages/plugin-image/web/src/image-viewer.jsx:121]: don't forget to uncomment :)

Comment (oded-soffrin) [reply packages/plugin-image/web/src/image-viewer.jsx:121]: I'm not going to merge this branch, just wanted to test if it looks ok in all fixtures

**Pull request closed without merging**